smoke-tests/ember-test-app/tests/integration/strict-mode-test.js

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { precompileTemplate } from '@ember/template-compilation';
+
+import service from 'ember-test-app/helpers/service';
+
+module('tests/integration/components/gjs', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it works with ember helpers', async function (assert) {
```

Comment:
I added a whole app for just this tset because I couldn't find any other place where I'd have access to all the thing that this test requires.

If such a place exists, I'm happy to move this test there.
I think having a full app though to make sure things are roughly still wired up ok is good